background.js

```diff
@@ -1,4 +1,13 @@
 chrome.browserAction.onClicked.addListener(function (tab) {
-    chrome.tabs.executeScript(null, { file: "jquery-2.2.min.js" });
-    chrome.tabs.executeScript(null, { file: "main.js" });
+    chrome.tabs.getSelected(null, function (tab) {
+        var url = new URL(tab.url);
+        chrome.tabs.create({"url": url.origin});
```

Comment:
is it ok to open only `origin` url in new tab (it will be only domain)? For ex. if you want to use extension on this page `https://github.com/alexkorsun/header-highlighter-chrome-extension/pull/1/files` and you click on extension, it will open new tab with `https://github.com` url.